app/components/user_menu/signed_in_component.html.erb

```diff
@@ -1,11 +1,12 @@
 <div class="absolute inset-y-0 right-0 flex items-center pr-2 sm:static sm:inset-auto sm:ml-6 sm:pr-0">
-  <div class="text-white px-2" style="position:relative">
```

Comment:
Is it okay to change this from an inline style to a Tailwind style? I couldn't tell if the original was intentional.

Reply:
Huh, me neither. I don't see why not – go for it.